### DISCUSSION TITLE
Connect to eth0 from the host machine

### DISCUSSION THREAD

**Discussion by mirza-ali-ctct** [rancher-sandbox/rancher-desktop#4158] [Q&A]
I am using Rancher Desktop `1.7.0` on Mac `Monterey`. For the service=LoadBalancer, I see we get the VM `etho0` address but that address is not reachable from the host machine(MacOS). Is there a way to allow host networking similar to `minikube`?

**Comment by jandubois** [comment]
You should get a bridged IP address if you enable "administrative access" in your settings. Getting an external interface from VMNET requires running a daemon as `root`.

The address comes from the DHCP server on your local network, so should be reachable from other local machines as well.
